docs/index.md

```diff
@@ -0,0 +1,89 @@
+<p align="center">
+<img src="images/zarhus-logo.png" width="300" align=center/>
+</p>
+
+# Welcome to the world of Zarhus - Your Digital Chess Knight 👋
+
+Inspired by the elegance and strategic prowess of the "huszár", meaning knight
+in chess (Polish: skoczek), Zarhus brings a unique blend of tradition and
+innovation to your embedded systems experience.
+
+## ❓ Why choose Zarhus
+
+### 🛡️ Deeper Meaning, Tactical Innovation:
+
+Delve into the world of chess analogies with Zarhus. Our name is more than just
+a label; it's a symbol of strategic brilliance and tactical adaptability,
+mirroring the knight's unconventional moves on the chessboard. Just as the
+knight navigates through complex scenarios, Zarhus operates seamlessly in the
+hardware integration area, showcasing adaptability and versatility.
+
+### 💡 Innovation at Every Move:
+
+Zarhus isn't just an operating system; it's your digital chess knight, making
+moves that matter. Experience the innovation that comes with a system designed
+to adapt, secure, and conquer the complexities of the ever-evolving embedded
+solutions.
+
+### 🌐 Your Digital Landscape, Secured:
+
+Just as the knight secures the chessboard, Zarhus safeguards your embedded
+systems. Navigate through the intricate ecosystem of hardware and software with
+ease, confident in the adaptability and resilience that Zarhus provides.
```

Comment:
Over time we would look for more practical, factual and number driven reasons. Let's say it is OK for now now despite little BLIPpy